openfisca_france/parameters/prelevements_sociaux/professions_liberales/auto_entrepreneur/cotisations_prestations/service.yaml

```
@@ -52,6 +58,21 @@ metadata:
     2022-10-01:
       title: Décret n° 2022-1529 du 7 décembre 2022
       href: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000046710847
+    2024-07-01:
+      - title: Décret n° 2024-484 du 30 mai 2024 - art. 1
+        href: https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000049622560/2024-06-01
```

```suggestion
      - title: Décret n° 2024-484 du 30 mai 2024 - art. 1
        href: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000049620959
```
Pour mettre la version initiale du décret